test/test-trace-api.js

```diff
@@ -94,13 +101,14 @@ describe('Trace Interface', function() {
     
     before(function() {
       traceAPI.enable_(agent);
-      common.init(traceAPI);
-      common.avoidTraceWriterAuth(traceAPI);
+      traceAPI.private_().traceWriter.request = request;
+      common.avoidTraceWriterAuth();
     });
 
     afterEach(function() {
-      common.cleanTraces(traceAPI);
-      common.clearNamespace(traceAPI);
+      TraceWriter.get().buffer_ = []; // clear traces
```

Comment:
Clearing the buffer can be expressed with `cleanTraces`; I've made small changes to make this possible.

Since I removed the ability to specify `traceAPI` for `clearNamespace`, there isn't a way for `clearNamespace` to set the `namespace` property on `traceAPI`.